bucket/nuwen-gcc.json

```
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://nuwen.net/mingw.html",
+    "description": "Nuwen's MinGW Distro with GCC, Boost etc.",
```

```suggestion
    "description": "Nuwen's MinGW Distro with coreutils, 7zip and extra headers",
```

Also `nuwen-gcc.json` -> `nuwen-mingw-without-git.json` as per @chawyehsu's suggestion